bst/bst_code.rb

```diff
@@ -0,0 +1,360 @@
+=begin
```

Comment:
file structure can be improved:
/bst/bst_node.rb
/bst/src/bst.rb
/bst/test/input.txt